ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -565,3 +565,18 @@ def check_capacity_breakdown(self, project_name, pod_name):
             logger.error(f"The pod {pod_name} not found on capacity_breakdown")
             res = False
         return res
+
+    def validate_unprivileged_access(self):
+        """
+        Function to verify the unprivileged users can't access ODF dashbaord
+        """
+        self.do_click(self.validation_loc["developer_dropdown"])
+        self.do_click(self.validation_loc["select_administrator"], timeout=5)
+        try:
+            self.navigate_odf_overview_page()
+        except Exception:
```

Comment:
Can we raise any specific exception for this case?

Reply:
Here I expect `TimeoutException` from do_click() method inside self.navigate_odf_overview_page() but whats weird is if I use below piece of code to have specific exception handling

>         except TimeoutException:
            logger.info(
                "As expected, ODF dashboard is not available for the unprivileged user"
            )

I see that exception is not caught even though raised exception is TimeoutException. It only works when I keep the exception handling broader by using just Exception clause